nautobot/circuits/models.py

```diff
@@ -317,6 +317,9 @@ class CircuitTermination(PrimaryModel, PathEndpoint, CableTermination):
     xconnect_id = models.CharField(max_length=50, blank=True, verbose_name="Cross-connect ID")
     pp_info = models.CharField(max_length=100, blank=True, verbose_name="Patch panel/port(s)")
     description = models.CharField(max_length=200, blank=True)
+    _path = models.ForeignKey(
+        to="dcim.CablePath", on_delete=models.SET_NULL, null=True, blank=True, related_name="circuit_terminations"
+    )
```

Comment:
Could we change this on the abstract base class `PathEndpoint` to the below reverse relation name? We need to document that as the standard for abstract base classes.

```py
related_name="%(app_label)s_%(class)s_related"
```